docs/user/user/pings/baseline.md

```diff
@@ -4,7 +4,11 @@
 
 This ping is intended to provide metrics that are managed by the Glean SDK itself, and not explicitly set by the application or included in the application's `metrics.yaml` file.
 
-> **Note:** As the `baseline` ping was specifically designed for mobile operating systems, it is not sent when using the Glean Python bindings.
+{{#include ../../../shared/blockquote-warning.html}}
+
+##### This ping is not available in Python and Glean.js
```

Comment:
I wonder if it would be better to add a "Available on" section at the top of each ping page.

Reply:
Do you prefer I do this in addition to the table in the 'sent-by-glean.md' document?

Reply:
Yes :)